node/network/bridge/src/lib.rs

```diff
@@ -185,18 +190,29 @@ impl Network for Arc<sc_network::NetworkService<Block, Hash>> {
 
 		Box::pin(ActionSink(&**self))
 	}
+
+	fn set_priority_group(&self, group_id: String, multiaddresses: HashSet<Multiaddr>) -> Result<(), String> {
+		sc_network::NetworkService::set_priority_group(&**self, group_id, multiaddresses)
+	}
 }
 
 /// The network bridge subsystem.
-pub struct NetworkBridge<N>(N);
+// TODO: does it need to be generic over AuthorityDiscoveryService?
```

Comment:
No, not necessary

Reply:
What I had in mind is instead to do `impl Network for (Arc<NetworkService>, AuthorityDiscoveryService)`.
It doesn't really make sense to be to be generic over the `NetworkService` but not the `AuthorityDiscoveryService`. It's either both or none.

Reply:
Yeah, I agree, it doesn't make sense to abstract over `NetworkService`, but not `AuthorityDiscoveryService`.